pkg/ratelimit/config.go

```diff
@@ -77,15 +90,25 @@ func New(ctx context.Context, conf config.RateLimiting, blobConf config.BlobConf
 
 var errInvalidRate = errors.DefineInvalidArgument("invalid_rate", "invalid rate `{rate}` for profile `{name}`")
 
+func newStore(conf StoreConfig) (store throttled.GCRAStoreCtx, err error) {
+	switch conf.Provider {
+	case "redis":
+		return redisstore.NewCtx(conf.Redis, conf.Redis.Key(""))
```

Comment:
Why is this an empty key?

Reply:
It's basically a prefix - see [this discussion](https://github.com/TheThingsIndustries/lorawan-stack/pull/4071#discussion_r1481131592).